docs/playlist/soundtrack.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebarDepth: 2
+title: Soundtrack
+---
+
+| Playlist Name | Playlist Link | Platform | Playlist Author Name/Contributor Name | Note | Subgenre |
+| ------------- |-------------- | -------- | ------------------------------------- | ---- | -------- |
+| Assassin's Creed Valhalla (Soundtrack) | [YouTube Music] https://music.youtube.com/playlist?list=PLDisKgcnAC4RbwywMTbJpC-uBg6mUeCsO&feature=share | YouTube Music | [Parth Kulkarni] (https://github.com/parthkgh24) | - | - |
+|FIFA 2022 Soundtrack | [Spotify] https://open.spotify.com/playlist/2a6OSlFchVcDonrXMMJ6EM?si=2fc2db2d922d4623 | Spotify | [Parth Kulkarni] (https://github.com/parthkgh24) | - | - |
+| Lucifer Soundtrack | [YouTube Music] https://music.youtube.com/playlist?list=PLlO3ranUZrZ4o9oyhU51heXdnRF2SY9EU&feature=share | YouTube Music | [Parth Kulkarni] (https://github.com/parthkgh24) | - | - |
+|Popular Movie Soundtracks | [Spotify] https://open.spotify.com/playlist/1VHaEzv3qjaDBAVJrEGEge?si=00564258b11a4da7 | Spotify | [Parth Kulkarni] (https://github.com/parthkgh24) | - | - |
+|Soundtracks from Superhero Movies | [YouTube Music] https://music.youtube.com/playlist?list=RDCLAK5uy_lJlBtxBu4iJF7YPUcnDxnQlrkYxZWCzwE&feature=share&playnext=1 | YouTube Music | [Parth Kulkarni] (https://github.com/parthkgh24) | - | - |
+| Star Wars Soundtracks | [Spotify] https://open.spotify.com/playlist/0yDagEjoveJC0dVWBJecU2?si=1f16b0db590b4f63 | Spotify | [Parth Kulkarni] (https://github.com/parthkgh24) | - | - |
+|Studio Ghibli Soundtracks | [Spotify] https://open.spotify.com/playlist/636JPjFfxIumqg3UQALZpY?si=2519381d393c47cf | Spotify | [Parth Kulkarni] (https://github.com/parthkgh24) | - | - |
```

Comment:
In playlist list and playlist author name section, could you change the format like this? for example `[Youtube Music](https://music.youtube.com/playlist?list=PLDisKgcnAC4RbwywMTbJpC-uBg6mUeCsO&feature=share)` and `[Parth Kulkarni](https://github.com/parthkgh24)`. Don't use space if you want to make a hyperlink in markdown.